.github/workflows/release.yml

```diff
@@ -200,8 +200,29 @@ jobs:
       base64-subjects: "${{ needs.sbom-assests.outputs.hashes }}"
       provenance-name: "karmada-sbom.intoto.jsonl"
       upload-assets: true
+
+  get-stable-tag:
+    name: Get Stable tag
+    runs-on: ubuntu-22.04
+    env:
+      GH_TOKEN: ${{ github.token }}
+    outputs:
+      stableTag: ${{ steps.get.outputs.stableTag }}
+    steps:
+    - name: get
+      id: get
+      run: |  
+        # get the latest version
+        export STABLE_TAG=`gh api repos/karmada-io/karmada/releases/latest | jq -r '.tag_name'`
+        echo "Got the stabe tag:$STABLE_TAG"
+        echo "event.tag:"${{ github.event.release.tag_name }}
+        echo "stableTag=$STABLE_TAG" >> "$GITHUB_OUTPUT"
+
   update-krew-index:
-    needs: release-assests
+    needs: 
+    - release-assests
+    - get-stable-tag
+    if: needs.get-stable-tag.outputs.stableTag == github.event.release.tag_name
```

Comment:
Is it possible to get the latest tag at the `update-krew-index` job? So that we don't need another job.

Reply:
krew-release-bot action don't care the version  https://github.com/karmada-io/karmada/pull/5224#discussion_r1735732558

It is possible to do this using the krew-release-bot binary instead of the krew-release-bot action https://github.com/rajatjindal/krew-release-bot/blob/ad6e92e6daadabfedaa107ce5186a82d41229ef1/examples/circleci.yml#L17-L18, but that means we'll be dropping the automatic updates for rajatjindal/krew-release-bot action, I'd update it if that's what you want.
But I'd go with adding a job, it's not a big deal.

Reply:
We can wait and see how upstream responds.

Reply:
I didn't mean to hack anything of `rajatjindal/krew-release-bot`, I mean adding an extra step to the `update-krew-index` job, so that we don't need to share the variable between jobs.

Reply:
Thanks for your clarify, i have update it , add a step and not a new job.

NOTE: have not test yet.